lib/rucio/core/replica.py

```diff
@@ -799,6 +808,48 @@ def _list_replicas_for_files(file_clause, state_clause, files, rse_clause, updat
             {'scope': scope, 'name': name} in files and files.remove({'scope': scope, 'name': name})
 
 
+def get_vp_endpoint():
+    vp_endpoint = config_get('virtual_placement', 'vp_endpoint', default='')
+    print('vp endpoint: ', vp_endpoint)
+    return vp_endpoint
+
+
+def get_multi_cache_prefix(cache_site, filename, logger=logging.log):
+
+    vp_endpoint = get_vp_endpoint()
+    if not vp_endpoint:
+        return ''
+
+    # print('Looking up prefix for cache:', cache_site, 'and filename:', filename)
+    x_caches = REGION.get('CacheSites')
+    if x_caches is NO_VALUE:
+        logger(logging.DEBUG, 'reloading xcache sites and ranges.')
```

Comment:
this is not useful for debugging and just indicates normal operation

Reply:
true. removed.